php/class-coauthors-guest-authors.php

```diff
@@ -639,7 +639,7 @@ function metabox_manage_guest_author_contact_info() {
 			$pm_key = $this->get_post_meta_key( $field['key'] );
 			$value = get_post_meta( $post->ID, $pm_key, true );
 			echo '<tr><th>';
-			echo '<label for="' . esc_attr( $pm_key ) . '">' . $field['label'] . '</label>';
+			echo '<label for="' . esc_attr( $pm_key . '">' . $field['label'] ) . '</label>';
```

Comment:
`esc_attr` should stay as-is. `$field['label']` can probably be wrapped in `esc_html`.